var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -20,6 +22,8 @@ class Ncurses(AutotoolsPackage, GNUMirrorPackage):
     # URL must remain http:// so Spack can bootstrap curl
     gnu_mirror_path = "ncurses/ncurses-6.1.tar.gz"
 
+    executables = [r'ncurses\d*-config']
```

Comment:
```suggestion
    executables = [r'^ncursesw?\d*-config$']
```
My Spack installation has both `ncurses6-config` and `ncursesw6-config`